### DISCUSSION TITLE
Restructure PIDName

### DISCUSSION THREAD

**Discussion by Sun-flow** [AllenNeuralDynamics/aind-data-schema#898] [General]
This is a low prio issue, but wanted to open a conversation about the `PIDName` structure. 

It seems to work weirdly with some cases, due to being a generic structure that covers a lot of different applications (source, research registry/database used, info about antibody/reagent).

We should maybe consider splitting this class into 2 subclasses that get combined for more organized/named/clear structures at the usage level.

Current state: 
![image](https://github.com/AllenNeuralDynamics/aind-data-schema/assets/18150105/c5405d8d-199f-4a43-a52f-2022aa977232)

Proposed states for consideration:
![image](https://github.com/AllenNeuralDynamics/aind-data-schema/assets/18150105/b9079632-a710-4dd9-a83b-663a83648d85)

It just seems like some branched specificity would be good here. Secondary examples are hard to define, so interested in thoughts where other modifications might be useful.

**Comment by dyf** [comment]
@Sun-flow Can you be more clear about what is not working with the existing structure?
